websites/R/RTLplay/presence.ts

```diff
@@ -0,0 +1,423 @@
+			presenceData.smallImageText = (await strings).live;
+
+			if (time) {
+				if (
+					document.querySelector('[type="duration"]').textContent.includes("-")
+				) {
+					presenceData.endTimestamp = presence.getTimestamps(
+						0,
+						presence.timestampFromFormat(
+							document
+								.querySelector('[type="duration"]')
+								.textContent.replace("-", "")
+						)
+					)[1];
+				} else {
+					presenceData.endTimestamp = presence.getTimestamps(
+						presence.timestampFromFormat(
+							document.querySelector('[type="currentTime"]').textContent
+						),
+						presence.timestampFromFormat(
+							document.querySelector('[type="duration"]').textContent
+						)
+					)[1];
+				}
+			}
+
+			if (buttons) {
+				presenceData.buttons = [
+					{
+						label: (await strings).buttonWatchStream,
+						url: href, // We are not redirecting directly to the raw stream, it's only the livestream page
+					},
+				];
+			}
+
+			break;
+		}
+
+		/* Channel HUB page
+	Page HUB d'une chaîne
+	(ex: https://www.rtlplay.be/tvi) */
+		case pathname.split("/")[1] === "tvi" ||
+			pathname.split("/")[1] === "club" ||
+			pathname.split("/")[1] === "plug" ||
+			pathname.split("/")[1] === "bel" ||
+			pathname.split("/")[1] === "contact": {
+			presenceData.details = privacy
+				? (await strings).viewAPage
+				: (await strings).viewChannel;
+			presenceData.state = privacy ? "" : `${getChannel(pathname).channel}`;
+
+			presenceData.largeImageKey = getChannel(pathname).logo;
+			presenceData.smallImageKey = Assets.Reading;
+			presenceData.smallImageText = (await strings).browsing;
+
+			if (time) presenceData.startTimestamp = browsingTimestamp;
+
+			break;
+		}
+
+		/* Show, Serie, Movie page 
+	Page d'une émission, une série, un film
+	(ex: https://www.rtlplay.be/chicago-fire-p_8947) */
+		case document.querySelector("div.sc-gad7tv-0.enEPMo > span") !== null: {
+			presenceData.details = privacy
+				? (await strings).viewAPage
+				: `${(await strings).viewPage} ${document
+						.querySelector("div.sc-gad7tv-0.enEPMo > span")
+						.textContent.split("|")[0]
+						.trim()}`; // Content trimmed of tags line (ex: Série | Science-fiction | Drame)
+			presenceData.state = privacy
+				? ""
+				: getTitleTrimmed(document.querySelector("head > title").textContent);
+
+			presenceData.smallImageKey = Assets.Reading;
+			presenceData.smallImageText = (await strings).browsing;
+
+			if (time) presenceData.startTimestamp = browsingTimestamp;
+
+			if (buttons) {
+				presenceData.buttons = [
+					{
+						label: (await strings).viewAPage,
+						url: href,
+					},
+				];
+			}
+
+			break;
+		}
+
+		/* Category (Theme) page 
+	Page Catégorie (Thématique)
+	(ex: https://www.rtlplay.be/rtl_play/divertissement-rtl-play-f_405)*/
+		case document.querySelector("div > header > div > div > h1") !== null ||
+			document.querySelector(
+				"div > header > div > div > nav > div.sc-nqi6nw-0.cNDqJj > h1"
+			) !== null: {
```

Comment:
Try to shorten selectors. E.g.
```suggestion
			document.querySelector('[aria-current="page"]') !== null: {
```